setup.py

```diff
@@ -0,0 +1,15 @@
+from setuptools import setup
+
+setup(name='cromwell-tools',
+      version='1.0.0.dev1',
+      description='Utilities for interacting with the Cromwell workflow engine',
+      url='http://github.com/broadinstitute/cromwell-tools',
+      author='Mint Team',
+      author_email='mintteam@broadinstitute.org',
```

Comment:
Didn't realize we had this. Should we update our other packages to reflect this email?

Reply:
I think we should yeah. Better to make it clear the team is supporting the tool rather than an individual.